Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Pull and use the official Docker Hub Ubuntu 14.04 base image
+FROM ubuntu:14.04
+
+# Update the base image
```

Comment:
Not recommended by [dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/). Upstream layers should already be up to date.